docs/howto/hooks/lua.md

```diff
@@ -412,18 +412,18 @@ Parameters:
 
 A package used to export Delta Lake tables from lakeFS to an external cloud storage.
 
-### `lakefs/catalogexport/delta_exporter.export_delta_log(action, table_names, writer, delta_client, table_descriptors_path)`
+### `lakefs/catalogexport/delta_exporter.export_delta_log(action, table_def_names, write_object, delta_client, table_descriptors_path)`
 
 The function used to export Delta Lake tables.
 The return value is a table with mapping of table names to external table location (from which it is possible to query the data).
 
 Parameters:
 
 - `action`: The global action object
-- `table_names`: Delta tables name list (e.g. `{"table1", "table2"}`)
-- `writer`: A writer function with `function(bucket, key, data)` signature, used to write the exported Delta Log (e.g. `aws/s3.s3_client.put_object`)
+- `table_def_names`: Delta tables name list (e.g. `{"table1", "table2"}`)
+- `write_object`: A writer function with `function(bucket, key, data)` signature, used to write the exported Delta Log (e.g. `aws/client.s3_client.put_object` or `azure/client.client.put_object`)
```

Comment:
I'm not sure what `aws/client.s3_client.put_object`. 
If i understand correctly the authors intention (@Jonathan-Rosenberg ) it should be [aws/s3.put_object(bucket, key, value)](https://docs.lakefs.io/howto/hooks/lua.html#awss3put_objectbucket-key-value)

Reply:
Documentation had a lot of bugs - fixed